html/js/popper.min.js

```diff
@@ -1,6 +1,6 @@
 /**
```

Comment:
Maybe out of scope for this pr, but we really should load these via npm instead

Reply:
Yes, could be a very big JS cleanup.